internal/js/modules/k6/browser/common/frame.go

```diff
@@ -1002,6 +1002,58 @@ func (f *Frame) getAttribute(selector, name string, opts *FrameBaseOptions) (str
 	return s, true, nil
 }
 
+// Locator creates and returns a new locator for this frame.
+func (f *Frame) GetByRole(role string, opts *GetByRoleOptions) *Locator {
+	f.log.Debugf("Frame:Locator", "fid:%s furl:%q role:%q opts:%+v", f.ID(), f.URL(), role, opts)
```

Comment:
Should we refer to the method name instead, to keep it consistent with the other methods? 🤔 

```suggestion
	f.log.Debugf("Frame: GetByRole", "fid:%s furl:%q role:%q opts:%+v", f.ID(), f.URL(), role, opts)
```